### PR TITLE
Fix string highlighting breaking chat sanitization

### LINKED ISSUE #10854: Goonchat doesn't escape HTML properly
Say `<A HREF='BAD HREF STUFF= YES MASTER'>click me!!!</A>` out loud in-game
it werks

### PR DESCRIPTION
**Pull request by PJB3005**
Also makes goonchat use BYOND's JSON code.

Fixes #10854

### CODE REVIEW

**Comment by Exxion** [comment]
It would be great if we could completely remove all chat sanitization only on the test server, because fucking around with it is great but also potentially gamebreaking

File: goon/browserassets/js/browserOutput.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ function highlightTerms(el) {
 							break;
 						}
 					}
-					newText += newWord ? newWord : words[w];
+					newText += (newWord ? newWord : words[w]).replace("<", "&lt;");
```
Comment (9600bauds) [inline goon/browserassets/js/browserOutput.js:129]: Is there any way to do HTML stuff without using <?

**Comment by ComicIronic** [comment]
> BYOND has JSON code

Lummox's priority list is fucked.

File: goon/browserassets/js/browserOutput.js (hunk 1 above)
Comment (Exxion) [inline goon/browserassets/js/browserOutput.js:129]: You can do text entities, but the worst thing you can do with those is reverse text

**Comment by PJB3005** [comment]
I don't really see how it's fucked @ComicIronic, native BYOND JSON encoding is amazing.

**Pull request merged**